services/identity/index.ts

```diff
@@ -33,10 +40,16 @@ const totpGenerator = new TotpGenerator({
   expiry: parseInt(OTP_EXPIRY!, 10) ?? undefined,
 })
 
+if (!POSTMAN_API_KEY) {
+  throw new Error(
+    "Please ensure that you have set POSTMAN_API_KEY in your env vars and that you have sourced them!"
+  )
+}
+
```

Comment:
local dev runs with a mockMailer - we might want to put this warning only when it's not local_dev